templates/parent.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
```

Comment:
It's harmless, but is there a simple way to prevent the grunt task for generating this stub "parent" template?